---
title: Power calculations for detection of novel cell types
author: John Reid
output: html_document
layout: post
comment: true
references:
- id: PatelSinglecellRNAseqhighlights2014a
  type: article-journal
  author:
  - family: Patel
    given: Anoop P.
  - family: Tirosh
    given: Itay
  - family: Trombetta
    given: John J.
  - family: Shalek
    given: Alex K.
  - family: Gillespie
    given: Shawn M.
  - family: Wakimoto
    given: Hiroaki
  - family: Cahill
    given: Daniel P.
  - family: Nahed
    given: Brian V.
  - family: Curry
    given: William T.
  - family: Martuza
    given: Robert L.
  - family: Louis
    given: David N.
  - family: Rozenblatt-Rosen
    given: Orit
  - family: Suvà
    given: Mario L.
  - family: Regev
    given: Aviv
  - family: Bernstein
    given: Bradley E.
  issued:
  - year: '2014'
    month: '6'
    day: '20'
  accessed:
  - year: '2014'
    month: '9'
    day: '29'
  title: Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma
  container-title: Science
  container-title-short: Science
  page: '1396-1401'
  volume: '344'
  issue: '6190'
  abstract: Human cancers are complex ecosystems composed of cells with distinct phenotypes,
    genotypes, and epigenetic states, but current models do not adequately reflect
    tumor composition in patients. We used single-cell RNA sequencing (RNA-seq) to
    profile 430 cells from five primary glioblastomas, which we found to be inherently
    variable in their expression of diverse transcriptional programs related to oncogenic
    signaling, proliferation, complement/immune response, and hypoxia. We also observed
    a continuum of stemness-related expression states that enabled us to identify
    putative regulators of stemness in vivo. Finally, we show that established glioblastoma
    subtype classifiers are variably expressed across individual cells within a tumor
    and demonstrate the potential prognostic implications of such intratumoral heterogeneity.
    Thus, we reveal previously unappreciated heterogeneity in diverse regulatory programs
    central to glioblastoma biology, prognosis, and therapy. Cancer at single-cell
    resolution Single-cell sequencing can illuminate the genetic properties of brain
    cancers and reveal heterogeneity within a tumor. Patel et al. examined the genome
    sequence of single cells isolated from brain glioblastomas. The findings revealed
    shared chromosomal changes but also extensive transcription variation, including
    genes related to signaling, which represent potential therapeutic targets. The
    authors suggest that the variation in tumor cells reflects neural development
    and that such variation among cancer cells may prove to have clinical significance.
    Science, this issue p. 1396
  URL: http://www.sciencemag.org/content/344/6190/1396
  DOI: 10.1126/science.1254257
  ISSN: 0036-8075, 1095-9203
  language: en-US

- id: GrunSinglecellmessengerRNA2015
  type: article-journal
  author:
  - family: Grün
    given: Dominic
  - family: Lyubimova
    given: Anna
  - family: Kester
    given: Lennart
  - family: Wiebrands
    given: Kay
  - family: Basak
    given: Onur
  - family: Sasaki
    given: Nobuo
  - family: Clevers
    given: Hans
  - family: Oudenaarden
    given: Alexander
    non-dropping-particle: van
  issued:
  - year: '2015'
    month: '9'
    day: '10'
  accessed:
  - year: '2015'
    month: '9'
    day: '10'
  title: Single-cell messenger RNA sequencing reveals rare intestinal cell types
  container-title: Nature
  container-title-short: Nature
  page: '251-255'
  volume: '525'
  issue: '7568'
  abstract: Understanding the development and function of an organ requires the characterization
    of all of its cell types. Traditional methods for visualizing and isolating subpopulations
    of cells are based on messenger RNA or protein expression of only a few known
    marker genes. The unequivocal identification of a specific marker gene, however,
    poses a major challenge, particularly if this cell type is rare. Identifying rare
    cell types, such as stem cells, short-lived progenitors, cancer stem cells, or
    circulating tumour cells, is crucial to acquire a better understanding of normal
    or diseased tissue biology. To address this challenge we first sequenced the transcriptome
    of hundreds of randomly selected cells from mouse intestinal organoids, cultured
    self-organizing epithelial structures that contain all cell lineages of the mammalian
    intestine. Organoid buds, like intestinal crypts, harbour stem cells that continuously
    differentiate into a variety of cell types, occurring at widely different abundances.
    Since available computational methods can only resolve more abundant cell types,
    we developed RaceID, an algorithm for rare cell type identification in complex
    populations of single cells. We demonstrate that this algorithm can resolve cell
    types represented by only a single cell in a population of randomly sampled organoid
    cells. We use this algorithm to identify Reg4 as a novel marker for enteroendocrine
    cells, a rare population of hormone-producing intestinal cells. Next, we use Reg4
    expression to enrich for these rare cells and investigate the heterogeneity within
    this population. RaceID confirmed the existence of known enteroendocrine lineages,
    and moreover discovered novel subtypes, which we subsequently validated in vivo.
    Having validated RaceID we then applied the algorithm to ex vivo-isolated Lgr5-positive
    stem cells and their direct progeny. We find that Lgr5-positive cells represent
    a homogenous abundant population of stem cells mixed with a rare population of
    Lgr5-positive secretory cells. We envision broad applicability of our method for
    discovering rare cell types and the corresponding marker genes in healthy and
    diseased organs.
  keyword: Cellular noise,Computational models,Gene expression analysis
  URL: http://libsta28.lib.cam.ac.uk:2071/nature/journal/v525/n7568/full/nature14966.html
  DOI: 10.1038/nature14966
  ISSN: '0028-0836'
  language: en-US

- id: ZeiselCelltypesmouse2015
  type: article-journal
  author:
  - family: Zeisel
    given: Amit
  - family: Muñoz-Manchado
    given: Ana B.
  - family: Codeluppi
    given: Simone
  - family: Lönnerberg
    given: Peter
  - family: Manno
    given: Gioele La
  - family: Juréus
    given: Anna
  - family: Marques
    given: Sueli
  - family: Munguba
    given: Hermany
  - family: He
    given: Liqun
  - family: Betsholtz
    given: Christer
  - family: Rolny
    given: Charlotte
  - family: Castelo-Branco
    given: Gonçalo
  - family: Hjerling-Leffler
    given: Jens
  - family: Linnarsson
    given: Sten
  issued:
  - year: '2015'
    month: '6'
    day: '3'
  accessed:
  - year: '2015'
    month: '3'
    day: '5'
  title: Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq
  container-title: Science
  container-title-short: Science
  page: '1138-1142'
  volume: '347'
  issue: '6226'
  abstract: The mammalian cerebral cortex supports cognitive functions such as sensorimotor
    integration, memory, and social behaviors. Normal brain function relies on a diverse
    set of differentiated cell types, including neurons, glia, and vasculature. Here,
    we have used large-scale single-cell RNA sequencing (RNA-seq) to classify cells
    in the mouse somatosensory cortex and hippocampal CA1 region. We found 47 molecularly
    distinct subclasses, comprising all known major cell types in the cortex. We identified
    numerous marker genes, which allowed alignment with known cell types, morphology,
    and location. We found a layer I interneuron expressing Pax6 and a distinct postmitotic
    oligodendrocyte subclass marked by Itpr2. Across the diversity of cortical cell
    types, transcription factors formed a complex, layered regulatory code, suggesting
    a mechanism for the maintenance of adult cell type identity. Cellular diversity
    in the brain revealed The mammalian brain has an extraordinarily large number
    of cells. Although there are quite a few different cell types, many cells in any
    one category tend to look alike. Zeisel et al. analyzed the transcriptomes of
    mouse brain cells to reveal more than meets the eye. Interneurons of similar type
    were found in dissimilar regions of the brain. Oligodendrocytes that seemed to
    be all of one class were differentiated by their molecular signatures into a half-dozen
    classes. Microglia associated with blood vessels were distinguished from look-alike
    perivascular macrophages. Thus, the complex microanatomy of the brain can be revealed
    by the RNAs expressed in its cells. Science, this issue p. 1138
  URL: http://www.sciencemag.org/content/347/6226/1138
  DOI: 10.1126/science.aaa1934
  ISSN: 0036-8075, 1095-9203
  language: en-US

- id: BacherDesigncomputationalanalysis2016
  type: article-journal
  author:
  - family: Bacher
    given: Rhonda
  - family: Kendziorski
    given: Christina
  issued:
  - year: '2016'
  accessed:
  - year: '2016'
    month: '8'
    day: '10'
  title: Design and computational analysis of single-cell RNA-sequencing experiments
  container-title: Genome Biology
  container-title-short: Genome Biology
  page: '63'
  volume: '17'
  abstract: Single-cell RNA-sequencing (scRNA-seq) has emerged as a revolutionary
    tool that allows us to address scientific questions that eluded examination just
    a few years ago. With the advantages of scRNA-seq come computational challenges
    that are just beginning to be addressed. In this article, we highlight the computational
    methods available for the design and analysis of scRNA-seq experiments, their
    advantages and disadvantages in various settings, the open questions for which
    novel methods are needed, and expected future developments in this exciting area.
  URL: http://dx.doi.org/10.1186/s13059-016-0927-y
  DOI: 10.1186/s13059-016-0927-y
  ISSN: 1474-760X

- id: WagnerRevealingvectorscellular2016a
  type: article-journal
  author:
  - family: Wagner
    given: Allon
  - family: Regev
    given: Aviv
  - family: Yosef
    given: Nir
  issued:
  - year: '2016'
    month: '11'
  accessed:
  - year: '2017'
    month: '1'
    day: '27'
  title: Revealing the vectors of cellular identity with single-cell genomics
  container-title: Nature Biotechnology
  container-title-short: Nat Biotech
  page: '1145-1160'
  volume: '34'
  issue: '11'
  abstract: Single-cell genomics has now made it possible to create a comprehensive
    atlas of human cells. At the same time, it has reopened definitions of a cell’s
    identity and of the ways in which identity is regulated by the cell’s molecular
    circuitry. Emerging computational analysis methods, especially in single-cell
    RNA sequencing (scRNA-seq), have already begun to reveal, in a data-driven way,
    the diverse simultaneous facets of a cell’s identity, from discrete cell types
    to continuous dynamic transitions and spatial locations. These developments will
    eventually allow a cell to be represented as a superposition of ’basis vectors’,
    each determining a different (but possibly dependent) aspect of cellular organization
    and function. However, computational methods must also overcome considerable challenges—from
    handling technical noise and data scale to forming new abstractions of biology.
    As the scale of single-cell experiments continues to increase, new computational
    approaches will be essential for constructing and characterizing a reference map
    of cell identities.
  keyword: Cellular noise,Computer science,gene regulatory networks,Statistical methods,transcriptomics
  URL: http://www.nature.com/nbt/journal/v34/n11/full/nbt.3711.html
  DOI: 10.1038/nbt.3711
  ISSN: '1087-0156'
  language: en-US

- id: SvenssonPoweranalysissinglecell2017
  type: article-journal
  author:
  - family: Svensson
    given: Valentine
  - family: Natarajan
    given: Kedar Nath
  - family: Ly
    given: Lam-Ha
  - family: Miragaia
    given: Ricardo J.
  - family: Labalette
    given: Charlotte
  - family: Macaulay
    given: Iain C.
  - family: Cvejic
    given: Ana
  - family: Teichmann
    given: Sarah A.
  issued:
  - year: '2017'
    month: '3'
    day: '6'
  accessed:
  - year: '2017'
    month: '3'
    day: '8'
  title: Power analysis of single-cell RNA-sequencing experiments
  container-title: Nature Methods
  container-title-short: Nat Meth
  volume: advance online publication
  abstract: Single-cell RNA sequencing (scRNA-seq) has become an established and powerful
    method to investigate transcriptomic cell-to-cell variation, thereby revealing
    new cell types and providing insights into developmental processes and transcriptional
    stochasticity. A key question is how the variety of available protocols compare
    in terms of their ability to detect and accurately quantify gene expression. Here,
    we assessed the protocol sensitivity and accuracy of many published data sets,
    on the basis of spike-in standards and uniform data processing. For our workflow,
    we developed a flexible tool for counting the number of unique molecular identifiers
    (https://github.com/vals/umis/). We compared 15 protocols computationally and
    4 protocols experimentally for batch-matched cell populations, in addition to
    investigating the effects of spike-in molecular degradation. Our analysis provides
    an integrated framework for comparing scRNA-seq protocols.
  keyword: Gene expression,RNA sequencing,transcriptomics
  URL: http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4220.html
  DOI: 10.1038/nmeth.4220
  ISSN: '1548-7091'
  language: en-US

- id: SegerstolpeSingleCellTranscriptomeProfiling2016
  type: article-journal
  author:
  - family: Segerstolpe
    given: Åsa
  - family: Palasantza
    given: Athanasia
  - family: Eliasson
    given: Pernilla
  - family: Andersson
    given: Eva-Marie
  - family: Andréasson
    given: Anne-Christine
  - family: Sun
    given: Xiaoyan
  - family: Picelli
    given: Simone
  - family: Sabirsh
    given: Alan
  - family: Clausen
    given: Maryam
  - family: Bjursell
    given: Magnus K.
  - family: Smith
    given: David M.
  - family: Kasper
    given: Maria
  - family: Ämmälä
    given: Carina
  - family: Sandberg
    given: Rickard
  issued:
  - year: '2016'
    month: '10'
    day: '11'
  accessed:
  - year: '2017'
    month: '11'
    day: '10'
  title: Single-Cell Transcriptome Profiling of Human Pancreatic Islets in Health
    and Type 2 Diabetes
  container-title: Cell Metabolism
  container-title-short: Cell Metabolism
  page: '593-607'
  volume: '24'
  issue: '4'
  URL: http://www.cell.com/cell-metabolism/abstract/S1550-4131(16)30436-3
  DOI: 10.1016/j.cmet.2016.08.020
  ISSN: '1550-4131'
  language: en-US

- id: ViethpowsimRpoweranalysis2017
  type: article-journal
  author:
  - family: Vieth
    given: Beate
  - family: Ziegenhain
    given: Christoph
  - family: Parekh
    given: Swati
  - family: Enard
    given: Wolfgang
  - family: Hellmann
    given: Ines
  issued:
  - year: '2017'
    month: '11'
    day: '1'
  accessed:
  - year: '2017'
    month: '11'
    day: '21'
  title: 'powsimR: Power analysis for bulk and single cell RNA-seq experiments'
  title-short: powsimR
  container-title: Bioinformatics
  container-title-short: Bioinformatics
  page: '3486-3488'
  volume: '33'
  issue: '21'
  abstract: SummaryPower analysis is essential to optimize the design of RNA-seq experiments
    and to assess and compare the power to detect differentially expressed genes in
    RNA-seq data. PowsimR is a flexible tool to simulate and evaluate differential
    expression from bulk and especially single-cell RNA-seq data making it suitable
    for a priori and posterior power analyses.Availability and implementationThe R
    package and associated tutorial are freely available at https://github.com/bvieth/powsimR.Contactvieth@bio.lmu.de
    or hellmann@bio.lmu.deSupplementary informationSupplementary data are available
    at Bioinformatics online.
  URL: https://academic.oup.com/bioinformatics/article/33/21/3486/3952669
  DOI: 10.1093/bioinformatics/btx435
  ISSN: '1367-4803'

- id: KleinDropletBarcodingSingleCell2015
  type: article-journal
  author:
  - family: Klein
    given: Allon M.
  - family: Mazutis
    given: Linas
  - family: Akartuna
    given: Ilke
  - family: Tallapragada
    given: Naren
  - family: Veres
    given: Adrian
  - family: Li
    given: Victor
  - family: Peshkin
    given: Leonid
  - family: Weitz
    given: David A.
  - family: Kirschner
    given: Marc W.
  issued:
  - year: '2015'
    month: '5'
  accessed:
  - year: '2017'
    month: '11'
    day: '21'
  title: Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem
    Cells
  container-title: Cell
  page: '1187-1201'
  volume: '161'
  issue: '5'
  URL: http://linkinghub.elsevier.com/retrieve/pii/S0092867415005000
  DOI: 10.1016/j.cell.2015.04.044
  ISSN: '00928674'
  language: en-US

- id: DalMolinHowdesignsinglecell2018
  type: article-journal
  author:
  - family: Dal Molin
    given: Alessandra
  - family: Di Camillo
    given: Barbara
  issued:
  - year: '2018'
    month: '1'
    day: '31'
  title: 'How to design a single-cell RNA-sequencing experiment: Pitfalls, challenges
    and perspectives'
  title-short: How to design a single-cell RNA-sequencing experiment
  container-title: Briefings in Bioinformatics
  container-title-short: Brief. Bioinformatics
  abstract: The sequencing of the transcriptome of single cells, or single-cell RNA-sequencing,
    has now become the dominant technology for the identification of novel cell types
    in heterogeneous cell populations or for the study of stochastic gene expression.
    In recent years, various experimental methods and computational tools for analysing
    single-cell RNA-sequencing data have been proposed. However, most of them are
    tailored to different experimental designs or biological questions, and in many
    cases, their performance has not been benchmarked yet, thus increasing the difficulty
    for a researcher to choose the optimal single-cell transcriptome sequencing (scRNA-seq)
    experiment and analysis workflow. In this review, we aim to provide an overview
    of the current available experimental and computational methods developed to handle
    single-cell RNA-sequencing data and, based on their peculiarities, we suggest
    possible analysis frameworks depending on specific experimental designs. Together,
    we propose an evaluation of challenges and open questions and future perspectives
    in the field. In particular, we go through the different steps of scRNA-seq experimental
    protocols such as cell isolation, messenger RNA capture, reverse transcription,
    amplification and use of quantitative standards such as spike-ins and Unique Molecular
    Identifiers (UMIs). We then analyse the current methodological challenges related
    to preprocessing, alignment, quantification, normalization, batch effect correction
    and methods to control for confounding effects.
  keyword: normalization,single-cell RNA-seq,experimental design,experimental protocols,spike-ins,Unique
    Molecular Identifiers
  DOI: 10.1093/bib/bby007
  ISSN: '1477-4054'
  language: en-US

- id: ZiegenhainQuantitativesinglecelltranscriptomics2018
  type: article-journal
  author:
  - family: Ziegenhain
    given: Christoph
  - family: Vieth
    given: Beate
  - family: Parekh
    given: Swati
  - family: Hellmann
    given: Ines
  - family: Enard
    given: Wolfgang
  issued:
  - year: '2018'
    month: '3'
    day: '22'
  accessed:
  - year: '2018'
    month: '4'
    day: '4'
  title: Quantitative single-cell transcriptomics
  container-title: Briefings in Functional Genomics
  container-title-short: Brief Funct Genomics
  abstract: Single-cell RNA sequencing (scRNA-seq) is currently transforming our understanding
    of biology, as it is a powerful tool to resolve cellular heterogeneity and molecular
    networks. Over 50 protocols have been developed in recent years and also data
    processing and analyzes tools are evolving fast. Here, we review the basic principles
    underlying the different experimental protocols and how to benchmark them. We
    also review and compare the essential methods to process scRNA-seq data from mapping,
    filtering, normalization and batch corrections to basic differential expression
    analysis. We hope that this helps to choose appropriate experimental and computational
    methods for the research question at hand.
  URL: https://academic.oup.com/bfg/advance-article/doi/10.1093/bfgp/ely009/4951519
  DOI: 10.1093/bfgp/ely009
  language: en-US

- id: SagarHighThroughputSingleCellRNA2018
  type: chapter
  author:
  - family: Sagar
  - family: Herman
    given: Josip Stefan
  - family: Pospisilik
    given: John Andrew
  - family: Grün
    given: Dominic
  issued:
  - year: '2018'
  accessed:
  - year: '2018'
    month: '4'
    day: '5'
  title: High-Throughput Single-Cell RNA Sequencing and Data Analysis
  container-title: CpG Islands
  collection-title: Methods in molecular biology
  publisher: Humana Press, New York, NY
  page: '257-283'
  abstract: Understanding biological systems at a single cell resolution may reveal
    several novel insights which remain masked by the conventional population-based
    techniques providing an average readout of the behavior of cells. Single-cell
    transcriptome sequencing holds the potential to identify novel cell types and
    characterize the cellular composition of any organ or tissue in health and disease.
    Here, we describe a customized high-throughput protocol for single-cell RNA-sequencing
    (scRNA-seq) combining flow cytometry and a nanoliter-scale robotic system. Since
    scRNA-seq requires amplification of a low amount of endogenous cellular RNA, leading
    to substantial technical noise in the dataset, downstream data filtering and analysis
    require special care. Therefore, we also briefly describe in-house state-of-the-art
    data analysis algorithms developed to identify cellular subpopulations including
    rare cell types as well as to derive lineage trees by ordering the identified
    subpopulations of cells along the inferred differentiation trajectories.
  URL: https://link.springer.com/protocol/10.1007/978-1-4939-7768-0_15
  DOI: 10.1007/978-1-4939-7768-0_15
  ISBN: 978-1-4939-7767-3 978-1-4939-7768-0
  language: en-US

- id: HelmyRecombinantHumanInterleukin12014
  type: article-journal
  author:
  - family: Helmy
    given: Adel
  - family: Guilfoyle
    given: Mathew R
  - family: Carpenter
    given: Keri LH
  - family: Pickard
    given: John D
  - family: Menon
    given: David K
  - family: Hutchinson
    given: Peter J
  issued:
  - year: '2014'
    month: '5'
  accessed:
  - year: '2018'
    month: '4'
    day: '17'
  title: 'Recombinant Human Interleukin-1 Receptor Antagonist in Severe Traumatic
    Brain Injury: A Phase II Randomized Control Trial'
  title-short: Recombinant Human Interleukin-1 Receptor Antagonist in Severe Traumatic
    Brain Injury
  container-title: Journal of Cerebral Blood Flow & Metabolism
  page: '845-851'
  volume: '34'
  issue: '5'
  URL: http://journals.sagepub.com/doi/10.1038/jcbfm.2014.23
  DOI: 10.1038/jcbfm.2014.23
  ISSN: 0271-678X, 1559-7016
  language: en-US

- id: HelmyRecombinanthumaninterleukin12016
  type: article-journal
  author:
  - family: Helmy
    given: Adel
  - family: Guilfoyle
    given: Mathew R
  - family: Carpenter
    given: Keri LH
  - family: Pickard
    given: John D
  - family: Menon
    given: David K
  - family: Hutchinson
    given: Peter J
  issued:
  - year: '2016'
    month: '8'
  accessed:
  - year: '2018'
    month: '4'
    day: '17'
  title: Recombinant human interleukin-1 receptor antagonist promotes M1 microglia
    biased cytokines and chemokines following human traumatic brain injury
  container-title: Journal of Cerebral Blood Flow & Metabolism
  page: '1434-1448'
  volume: '36'
  issue: '8'
  abstract: Interleukin-1 receptor antagonist (IL1ra) has demonstrated efficacy in
    a wide range of animal models of neuronal injury. We have previously published
    a randomised controlled study of IL1ra in human severe TBI, with concomitant microdialysis
    and plasma sampling of 42 cytokines and chemokines. In this study, we have used
    partial least squares discriminant analysis to model the effects of drug administration
    and time following injury on the cytokine milieu within the injured brain. We
    demonstrate that treatment with rhIL1ra causes a brain-specific modification of
    the cytokine and chemokine response to injury, particularly in samples from the
    first 48 h following injury. The magnitude of this response is dependent on the
    concentration of IL1ra achieved in the brain extracellular space. Chemokines related
    to recruitment of macrophages from the plasma compartment (MCP-1) and biasing
    towards a M1 microglial phenotype (GM-CSF, IL1) are increased in patient samples
    in the rhIL1ra-treated patients. In control patients, cytokines and chemokines
    biased to a M2 microglia phenotype (IL4, IL10, MDC) are relatively increased.
    This pattern of response suggests that a simple classification of IL1ra as an
    ‘anti-inflammatory’ cytokine may not be appropriate and highlights the importance
    of the microglial response to injury.
  URL: http://journals.sagepub.com/doi/10.1177/0271678X15620204
  DOI: 10.1177/0271678X15620204
  ISSN: 0271-678X, 1559-7016
  language: en-US

- id: KimBraintraumaelicits2016
  type: article-journal
  author:
  - family: Kim
    given: Charles C.
  - family: Nakamura
    given: Mary C.
  - family: Hsieh
    given: Christine L.
  issued:
  - year: '2016'
    month: '12'
  accessed:
  - year: '2018'
    month: '4'
    day: '17'
  title: Brain trauma elicits non-canonical macrophage activation states
  container-title: Journal of Neuroinflammation
  volume: '13'
  issue: '1'
  abstract: 'Background: Macrophage polarization programs, commonly referred to
    as “classical” and “alternative” activation, are widely considered as distinct
    states that are exclusive of one another and are associated with different functions
    such as inflammation and wound healing, respectively. In a number of disease contexts,
    such as traumatic brain injury (TBI), macrophage polarization influences the extent
    of pathogenesis, and efforts are underway to eliminate pathogenic subsets. However,
    previous studies have not distinguished whether the simultaneous presence of both
    classical and alternative activation signatures represents the admixture of differentially
    polarized macrophages or if they have adopted a unique state characterized by
    components of both classical and alternative activation. Methods: We analyzed
    the gene expression profiles of individual monocyte-derived brain macrophages
    responding to TBI using single-cell RNA sequencing. RNA flow cytometry was used
    as another single-cell analysis technique to validate the single-cell RNA sequencing
    results. Results: The analysis of signature polarization genes by single-cell
    RNA sequencing revealed the presence of diverse activation states, including M(IL4),
    M(IL10), and M(LPS, IFNγ). However, the expression of a given polarization marker
    was no more likely than at random to predict simultaneous expression or repression
    of markers of another polarization program within the same cell, suggesting a
    lack of exclusivity in macrophage polarization states in vivo in TBI. Also unexpectedly,
    individual TBI macrophages simultaneously expressed high levels of signature polarization
    genes across two or three different polarization states and in several distinct
    and seemingly incompatible combinations. Conclusions: Single-cell gene expression
    profiling demonstrated that monocytic macrophages in TBI are not comprised of
    distinctly polarized subsets but are uniquely and broadly activated. TBI macrophage
    activation in vivo is deeply complex, with individual cells concurrently adopting
    both inflammatory and reparative features with a lack of exclusivity. These data
    provide physiologically relevant evidence that the early macrophage response to
    TBI is comprised of novel activation states that are discordant with the current
    paradigm of macrophage polarization—a key consideration for therapeutic modulation.'
  URL: http://jneuroinflammation.biomedcentral.com/articles/10.1186/s12974-016-0581-z
  DOI: 10.1186/s12974-016-0581-z
  ISSN: '1742-2094'
  language: en-US

- id: RansohoffpolarizingquestionM12016
  type: article-journal
  author:
  - family: Ransohoff
    given: Richard M
  issued:
  - year: '2016'
    month: '8'
  accessed:
  - year: '2018'
    month: '4'
    day: '17'
  title: 'A polarizing question: Do M1 and M2 microglia exist?'
  title-short: A polarizing question
  container-title: Nature Neuroscience
  page: '987-991'
  volume: '19'
  issue: '8'
  URL: http://www.nature.com/articles/nn.4338
  DOI: 10.1038/nn.4338
  ISSN: 1097-6256, 1546-1726
  language: en-US

- id: ZiegenhainComparativeAnalysisSingleCell2017
  type: article-journal
  author:
  - family: Ziegenhain
    given: Christoph
  - family: Vieth
    given: Beate
  - family: Parekh
    given: Swati
  - family: Reinius
    given: Björn
  - family: Guillaumet-Adkins
    given: Amy
  - family: Smets
    given: Martha
  - family: Leonhardt
    given: Heinrich
  - family: Heyn
    given: Holger
  - family: Hellmann
    given: Ines
  - family: Enard
    given: Wolfgang
  issued:
  - year: '2017'
    month: '2'
  accessed:
  - year: '2018'
    month: '6'
    day: '21'
  title: Comparative Analysis of Single-Cell RNA Sequencing Methods
  container-title: Molecular Cell
  page: '631-643.e4'
  volume: '65'
  issue: '4'
  abstract: 'Single-cell RNA sequencing (scRNA-seq) offers new possibilities to
    address biological and medical questions. However, systematic comparisons of the
    performance of diverse scRNA-seq protocols are lacking. We generated data from
    583 mouse embryonic stem cells to evaluate six prominent scRNA-seq methods: CEL-seq2,
    Drop-seq, MARS-seq, SCRBseq, Smart-seq, and Smart-seq2. While Smart-seq2 detected
    the most genes per cell and across cells, CEL-seq2, Drop-seq, MARS-seq, and SCRB-seq
    quantiﬁed mRNA levels with less ampliﬁcation noise due to the use of unique molecular
    identiﬁers (UMIs). Power simulations at different sequencing depths showed that
    Drop-seq is more cost-efﬁcient for transcriptome quantiﬁcation of large numbers
    of cells, while MARS-seq, SCRB-seq, and Smart-seq2 are more efﬁcient when analyzing
    fewer cells. Our quantitative comparison offers the basis for an informed choice
    among six prominent scRNA-seq methods, and it provides a framework for benchmarking
    further improvements of scRNA-seq protocols.'
  URL: http://linkinghub.elsevier.com/retrieve/pii/S1097276517300497
  DOI: 10.1016/j.molcel.2017.01.023
  ISSN: '10972765'
  language: en-US

- id: Montororevisedairwayepithelial2018
  type: article-journal
  author:
  - family: Montoro
    given: Daniel T.
  - family: Haber
    given: Adam L.
  - family: Biton
    given: Moshe
  - family: Vinarsky
    given: Vladimir
  - family: Lin
    given: Brian
  - family: Birket
    given: Susan E.
  - family: Yuan
    given: Feng
  - family: Chen
    given: Sijia
  - family: Leung
    given: Hui Min
  - family: Villoria
    given: Jorge
  - family: Rogel
    given: Noga
  - family: Burgin
    given: Grace
  - family: Tsankov
    given: Alexander M.
  - family: Waghray
    given: Avinash
  - family: Slyper
    given: Michal
  - family: Waldman
    given: Julia
  - family: Nguyen
    given: Lan
  - family: Dionne
    given: Danielle
  - family: Rozenblatt-Rosen
    given: Orit
  - family: Tata
    given: Purushothama Rao
  - family: Mou
    given: Hongmei
  - family: Shivaraju
    given: Manjunatha
  - family: Bihler
    given: Hermann
  - family: Mense
    given: Martin
  - family: Tearney
    given: Guillermo J.
  - family: Rowe
    given: Steven M.
  - family: Engelhardt
    given: John F.
  - family: Regev
    given: Aviv
  - family: Rajagopal
    given: Jayaraj
  issued:
  - year: '2018'
    month: '8'
    day: '1'
  accessed:
  - year: '2018'
    month: '8'
    day: '13'
  title: A revised airway epithelial hierarchy includes CFTR-expressing ionocytes
  container-title: Nature
  URL: http://www.nature.com/articles/s41586-018-0393-7
  DOI: 10.1038/s41586-018-0393-7
  ISSN: 0028-0836, 1476-4687
  language: en-US

- id: Plasschaertsinglecellatlasairway2018
  type: article-journal
  author:
  - family: Plasschaert
    given: Lindsey W.
  - family: Žilionis
    given: Rapolas
  - family: Choo-Wing
    given: Rayman
  - family: Savova
    given: Virginia
  - family: Knehr
    given: Judith
  - family: Roma
    given: Guglielmo
  - family: Klein
    given: Allon M.
  - family: Jaffe
    given: Aron B.
  issued:
  - year: '2018'
    month: '8'
    day: '1'
  accessed:
  - year: '2018'
    month: '8'
    day: '13'
  title: A single-cell atlas of the airway epithelium reveals the CFTR-rich pulmonary
    ionocyte
  container-title: Nature
  URL: http://www.nature.com/articles/s41586-018-0394-6
  DOI: 10.1038/s41586-018-0394-6
  ISSN: 0028-0836, 1476-4687
  language: en-US

- id: Haquepracticalguidesinglecell2017
  type: article-journal
  author:
  - family: Haque
    given: Ashraful
  - family: Engel
    given: Jessica
  - family: Teichmann
    given: Sarah A.
  - family: Lönnberg
    given: Tapio
  issued:
  - year: '2017'
    month: '8'
    day: '18'
  accessed:
  - year: '2018'
    month: '9'
    day: '3'
  title: A practical guide to single-cell RNA-sequencing for biomedical research and
    clinical applications
  container-title: Genome Medicine
  container-title-short: Genome Medicine
  page: '75'
  volume: '9'
  issue: '1'
  abstract: RNA sequencing (RNA-seq) is a genomic approach for the detection and quantitative
    analysis of messenger RNA molecules in a biological sample and is useful for studying
    cellular responses. RNA-seq has fueled much discovery and innovation in medicine
    over recent years. For practical reasons, the technique is usually conducted on
    samples comprising thousands to millions of cells. However, this has hindered
    direct assessment of the fundamental unit of biology—the cell. Since the first
    single-cell RNA-sequencing (scRNA-seq) study was published in 2009, many more
    have been conducted, mostly by specialist laboratories with unique skills in wet-lab
    single-cell genomics, bioinformatics, and computation. However, with the increasing
    commercial availability of scRNA-seq platforms, and the rapid ongoing maturation
    of bioinformatics approaches, a point has been reached where any biomedical researcher
    or clinician can use scRNA-seq to make exciting discoveries. In this review, we
    present a practical guide to help researchers design their first scRNA-seq studies,
    including introductory information on experimental hardware, protocol choice,
    quality control, data analysis and biological interpretation.
  URL: https://doi.org/10.1186/s13073-017-0467-4
  DOI: 10.1186/s13073-017-0467-4
  ISSN: 1756-994X

- id: BirnbaumPowerNumbersSingleCell2018
  type: article-journal
  author:
  - family: Birnbaum
    given: Kenneth D.
  issued:
  - year: '2018'
    month: '9'
    day: '7'
  title: 'Power in Numbers: Single-Cell RNA-Seq Strategies to Dissect Complex Tissues'
  title-short: Power in Numbers
  container-title: Annual Review of Genetics
  container-title-short: Annu. Rev. Genet.
  abstract: The growing scale and declining cost of single-cell RNA-sequencing (RNAseq)
    now permit a repetition of cell sampling that increases the power to detect rare
    cell states, reconstruct developmental trajectories, and measure phenotype in
    new terms such as cellular variance. The characterization of anatomy and developmental
    dynamics have not had an equivalent breakthrough since groundbreaking advances
    in live fluorescent microscopy. The new resolution obtained by single-cell RNA-seq
    is a boon to genetics because the novel description of phenotype offers the opportunity
    to refine gene function and dissect pleiotropy. In addition, the recent pairing
    of highthroughput genetic perturbation with single-cell RNA-seq has made practical
    a scale of genetic screening not previously possible. Expected final online publication
    date for the Annual Review of Genetics Volume 52 is November 23, 2018. Please
    see http://www.annualreviews.org/page/journal/pubdates for revised estimates.
  DOI: 10.1146/annurev-genet-120417-031247
  ISSN: '1545-2948'
  language: en-US

- id: PoplawskiFeasibilitysamplesize2018
  type: article-journal
  author:
  - family: Poplawski
    given: Alicia
  - family: Binder
    given: Harald
  issued:
  - year: '2018'
    month: '7'
    day: '20'
  accessed:
  - year: '2018'
    month: '9'
    day: '12'
  title: Feasibility of sample size calculation for RNA-seq studies
  container-title: Briefings in Bioinformatics
  container-title-short: Brief Bioinform
  page: '713-720'
  volume: '19'
  issue: '4'
  abstract: Abstract. Sample size calculation is a crucial step in study design but
    is not yet fully established for RNA sequencing (RNA-seq) analyses. To evaluate
    feasibi
  URL: https://academic.oup.com/bib/article/19/4/713/2920205
  DOI: 10.1093/bib/bbw144
  ISSN: '1467-5463'
  language: en-US

- id: ParekhzUMIsfastflexible2018
  type: article-journal
  author:
  - family: Parekh
    given: Swati
  - family: Ziegenhain
    given: Christoph
  - family: Vieth
    given: Beate
  - family: Enard
    given: Wolfgang
  - family: Hellmann
    given: Ines
  issued:
  - year: '2018'
    month: '6'
    day: '1'
  accessed:
  - year: '2018'
    month: '9'
    day: '12'
  title: zUMIs - A fast and flexible pipeline to process RNA sequencing data with
    UMIs
  container-title: GigaScience
  container-title-short: Gigascience
  volume: '7'
  issue: '6'
  abstract: AbstractBackground. Single-cell RNA-sequencing (scRNA-seq) experiments
    typically analyze hundreds or thousands of cells after amplification of the cDNA.
    The hi
  URL: https://academic.oup.com/gigascience/article/7/6/giy059/5005022
  DOI: 10.1093/gigascience/giy059
  language: en-US

- id: KumarUnderstandingdevelopmentstem2017
  type: article-journal
  author:
  - family: Kumar
    given: Pavithra
  - family: Tan
    given: Yuqi
  - family: Cahan
    given: Patrick
  issued:
  - year: '2017'
    month: '1'
    day: '1'
  accessed:
  - year: '2018'
    month: '9'
    day: '17'
  title: Understanding development and stem cells using single cell-based analyses
    of gene expression
  container-title: Development
  container-title-short: Development
  page: '17-32'
  volume: '144'
  issue: '1'
  abstract: Skip to Next Section In recent years, genome-wide profiling approaches
    have begun to uncover the molecular programs that drive developmental processes.
    In particular, technical advances that enable genome-wide profiling of thousands
    of individual cells have provided the tantalizing prospect of cataloging cell
    type diversity and developmental dynamics in a quantitative and comprehensive
    manner. Here, we review how single-cell RNA sequencing has provided key insights
    into mammalian developmental and stem cell biology, emphasizing the analytical
    approaches that are specific to studying gene expression in single cells.
  URL: http://dev.biologists.org/content/144/1/17
  DOI: 10.1242/dev.133058
  ISSN: 0950-1991, 1477-9129
  language: en-US

- id: AndrewsIdentifyingcellpopulations2018
  type: article-journal
  author:
  - family: Andrews
    given: Tallulah S.
  - family: Hemberg
    given: Martin
  issued:
  - year: '2018'
    month: '2'
    day: '1'
  accessed:
  - year: '2018'
    month: '9'
    day: '25'
  title: Identifying cell populations with scRNASeq
  container-title: Molecular Aspects of Medicine
  collection-title: The emerging field of single-cell analysis
  container-title-short: Molecular Aspects of Medicine
  page: '114-122'
  volume: '59'
  abstract: Single-cell RNASeq (scRNASeq) has emerged as a powerful method for quantifying
    the transcriptome of individual cells. However, the data from scRNASeq experiments
    is often both noisy and high dimensional, making the computational analysis non-trivial.
    Here we provide an overview of different experimental protocols and the most popular
    methods for facilitating the computational analysis. We focus on approaches for
    identifying biologically important genes, projecting data into lower dimensions
    and clustering data into putative cell-populations. Finally we discuss approaches
    to validation and biological interpretation of the identified cell-types or cell-states.
  URL: http://www.sciencedirect.com/science/article/pii/S0098299717300493
  DOI: 10.1016/j.mam.2017.07.002
  ISSN: '0098-2997'

- id: Campbellmolecularcensusarcuate2017
  type: article-journal
  author:
  - family: Campbell
    given: John N
  - family: Macosko
    given: Evan Z
  - family: Fenselau
    given: Henning
  - family: Pers
    given: Tune H
  - family: Lyubetskaya
    given: Anna
  - family: Tenen
    given: Danielle
  - family: Goldman
    given: Melissa
  - family: Verstegen
    given: Anne M J
  - family: Resch
    given: Jon M
  - family: McCarroll
    given: Steven A
  - family: Rosen
    given: Evan D
  - family: Lowell
    given: Bradford B
  - family: Tsai
    given: Linus T
  issued:
  - year: '2017'
    month: '3'
  accessed:
  - year: '2018'
    month: '9'
    day: '25'
  title: A molecular census of arcuate hypothalamus and median eminence cell types
  container-title: Nature Neuroscience
  page: '484-496'
  volume: '20'
  issue: '3'
  URL: http://www.nature.com/articles/nn.4495
  DOI: 10.1038/nn.4495
  ISSN: 1097-6256, 1546-1726
  language: en-US

- id: JiangGiniClustdetectingrare2016
  type: article-journal
  author:
  - family: Jiang
    given: Lan
  - family: Chen
    given: Huidong
  - family: Pinello
    given: Luca
  - family: Yuan
    given: Guo-Cheng
  issued:
  - year: '2016'
    month: '7'
    day: '1'
  accessed:
  - year: '2018'
    month: '9'
    day: '25'
  title: 'GiniClust: Detecting rare cell types from single-cell gene expression
    data with Gini index'
  title-short: GiniClust
  container-title: Genome Biology
  container-title-short: Genome Biology
  page: '144'
  volume: '17'
  issue: '1'
  abstract: High-throughput single-cell technologies have great potential to discover
    new cell types; however, it remains challenging to detect rare cell types that
    are distinct from a large population. We present a novel computational method,
    called GiniClust, to overcome this challenge. Validation against a benchmark dataset
    indicates that GiniClust achieves high sensitivity and specificity. Application
    of GiniClust to public single-cell RNA-seq datasets uncovers previously unrecognized
    rare cell types, including Zscan4-expressing cells within mouse embryonic stem
    cells and hemoglobin-expressing cells in the mouse cortex and hippocampus. GiniClust
    also correctly detects a small number of normal cells that are mixed in a cancer
    cell population.
  URL: https://doi.org/10.1186/s13059-016-1010-4
  DOI: 10.1186/s13059-016-1010-4
  ISSN: 1474-760X
...

Discovery of novel cell types
=============================

Here we discuss estimates of required sample sizes for the discovery of
novel cell types in single cell RNA-sequencing.

Theory
======

Power calculations for single cell experiments to discover novel cell
types are thin on the ground. We summarise some related work here.

-   Vieth et al. @ViethpowsimRpoweranalysis2017 provide some theory and
    an R package to power calculations in the differential
    expression context.

-   Haque et al. @Haquepracticalguidesinglecell2017 discuss practical
    issues associated with single cell experimental design. They discuss
    sample sizes required for novel cell type identification without
    developed a theory.

-   Dal Molin et al. @DalMolinHowdesignsinglecell2018 discuss how to
    design a single cell experiment to identify novel cell types but do
    not explicitly address sample size calculations.

-   Poplawski and Binder @PoplawskiFeasibilitysamplesize2018 provide an
    emprirical study of sample size requirements for differential
    expression in single cell data.

Applications
============

We summarise some studies (mainly sourced from Andrews and
Hemberg @AndrewsIdentifyingcellpopulations2018).

-   Grün et al. [@GrunSinglecellmessengerRNA2015] developed RaceID, an
    algorithm for detection of rare cell types. They studied single cell
    data from the intestine. The data set is in the [conquer
    database](http://imlspenticton.uzh.ch/robinson_lab/conquer/report-scater/GSE62270-GPL17021_scater.html).
    The data comprises 2,891 cells.

-   Jiang et al. @JiangGiniClustdetectingrare2016 developed GiniClust, a
    tool to detect novel cell types using the Gini index. They analysed
    publically available datasets @ZeiselCelltypesmouse2015
    [@KleinDropletBarcodingSingleCell2015; @PatelSinglecellRNAseqhighlights2014a].

-   Segerstolpe et al. @SegerstolpeSingleCellTranscriptomeProfiling2016
    analysed pancreatic cells of healthy and type II diabetic donors,
    revealing novel subpopulations.

-   Hypothalamic arcuate-median eminence complex: Campbell et
    al. @Campbellmolecularcensusarcuate2017 identified 50
    transcriptionally distinct cell populations.

-   Montoro et al. @Montororevisedairwayepithelial2018 and Plasschaert
    et al. @Plasschaertsinglecellatlasairway2018 discovered a novel
    airway cell type that expresses genes associated with
    cystic fibrosis.
